out/js/credit_card_form.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+var CreditCardForm = (function ($) {
```

Comment:
We might want to prefix global vars to avoid conflicts with core and other modules.

Reply:
👍